codec_specs.md

```diff
@@ -1131,13 +1131,13 @@ Block type name: Opaque data
 Description: the `BlockAdditional` data is interpreted as opaque additional data passed to the codec
 with the Block data. `BlockAddIDValue` **MUST** be 1.
 
-### ITU T.35 metadata
+### ITU T.35 Metadata
 
 Block type identifier: 4
 
 Block type name: ITU T.35 metadata
 
-Description: the `BlockAdditional` data is interpreted as ITU T.35 metadata, as defined by ITU-T T.35
+Description: the `BlockAdditional` data is interpreted as ITU T.35 metadata, as defined by [@!ITU-T.35]
```

Comment:
Probably not a non normative reference because it's not the actual data description of various data.